clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -2087,7 +2087,15 @@ protected DelegateConsumerRecordResult delegateConsumerRecord(
                 LeaderMetadataWrapper leaderMetadataWrapper =
                     new LeaderMetadataWrapper(consumerRecord.getOffset(), kafkaClusterId);
                 PubSubTopicPartition topicPartition = new PubSubTopicPartitionImpl(getVersionTopic(), subPartition);
-                veniceWriter.get().sendHeartbeat(topicPartition, callback, leaderMetadataWrapper);
+                // Leaders forward HB SOS message to local VT with updated isLeaderCompleted header
+                veniceWriter.get()
+                    .sendHeartbeat(
+                        topicPartition,
+                        callback,
+                        leaderMetadataWrapper,
+                        true,
```

Comment:
Nitpick: It might help readability to create a constant for that, `LEADER_IS_COMPLETE_HEADER = true`, or similar.

Reply:
Done